app/backend/src/clj/teet/contract/contract_commands.clj

```diff
@@ -32,8 +33,10 @@
          (company-db/business-registry-code-unique?
            db
            (company-model/company-business-registry-id-with-country-code form-data))]}
-  (let [company-fields (-> (select-keys form-data [:company/country :company/emails :company/phone-numbers
+  (let [company-fields (-> (select-keys form-data [:company/country
                                                    :company/business-registry-code :company/name])
+                           (assoc :company/emails (str/join "" (:company/emails form-data)))
+                           (assoc :company/phone-numbers (str/join "" (:company/phone-numbers form-data)))
```

Comment:
should these be joined by ", " instead of just "" ? Also `(update :company/emails #(str/join ", " %))` works a bit simpler.

Reply:
fixed by parsing vector of emails and vector of phone-numbers right after x-road query. And then read/wride as a single string